definitions/divorce/json/CaseEventToFields.json

```diff
@@ -7421,9 +7555,7 @@
     "PageLabel": "Review Acknowledgement of Service",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "No",
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
-    "RetriesTimeoutURLMidEvent": "120,120"
```

Comment:
@hosainnet again not sure if this should have been removed. if the callback is the one that generates the mini petition preview link I think it is used here